src/DurableTask.Netherite/Connections/ConnectionResolver.cs

```diff
@@ -58,5 +58,26 @@ public enum ResourceType
         /// <param name="storageChoice">The storage layer to use.</param>
         /// <param name="transportChoice">The transport layer to use.</param>
         public abstract void ResolveLayerConfiguration(string connectionName, out StorageChoices storageChoice, out TransportChoices transportChoice);
+
+        /// <summary>
+        /// Creates a connection resolver for a given connection string lookup function.
+        /// </summary>
+        /// <param name="connectionStringLookup">A function that maps connection names to connection strings.</param>
+        public static ConnectionResolver FromConnectionNameToConnectionStringResolver(Func<string, string> connectionStringLookup)
```

Comment:
Is there a more concise name we can use? We can at least drop `Resolver` suffix, as that is already expressed in the type ("ConnectionResolver"). For a more concise name, maybe `FromNamedConnectionString`?

Reply:
I think I prefer the verbose name here. I find it helpful if implementations of an abstract class share the last word with the superclass.